### DISCUSSION TITLE
Scheduling price changes

### DISCUSSION THREAD

**Discussion by dekoza** [saleor/saleor#11473] [General] [answered]
Assume I'm using Saleor Cloud.

I need a way to schedule price changes for my products. I know there is [sales feature](https://docs.saleor.io/docs/3.x/dashboard/discounts/sales) but its documentation suggests that it is usable only for short-lived price drops. I need a way to update price of my product(s) on a certain date - either lower of raise it. Should I use `sales` but with negative value if I want to raise the price? Maybe rename the feature from `sales` to `price change scheduling`? :smile: 

How could this be done in most effective manner?

**Comment by karolkielecki** [comment]
Hi @dekoza, what comes to my mind as a quick workaround, you can build an app that sends a mutation at a specific time to Saleor. Could you please create an issue/feature request? Cheers!

**Comment by timuric** [comment, marked as answer]
@dekoza it sounds like you are looking for actual price changes raher just discounts.

You would need to automate this logic using API.
One issue to keep in mind, fluctuting prices will affect users that are trying to checkout. If you want to fix the price of an item in the checkout object you could use this API https://docs.saleor.io/docs/3.x/developer/checkout#setting-custom-prices-for-checkout-items

Perhaps more details about your use case could help us to explore more solutions.
